contracts/CTKNCrowdsale.sol

```diff
@@ -11,10 +11,34 @@ import 'zeppelin-solidity/contracts/token/ERC20/MintableToken.sol';
 contract CTKNCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale {
     using SafeMath for uint256;
 
+    // the number of wei one US Dollar buys.
```

Comment:
Can we make the comment have a qualifier or reference a qualifying statement like "according to the ITO terms or crowdsale terms" or "according to the price at which we will buy Ether at end of crowdsale".

Reply:
It seems that we set the dollar rate on crowdsale launch so perhaps that would be enough of a qualifier

Reply:
We can write some acceptance these for this later in case we set a rate and the price of eth to dollar drops a lot or rises a lot.  I'm not sure if I've wrapped my head around this.  Originally I had understood that we could have some sort of contingency statements as well like if the price of Eth drops > 10% we will buy at a different rate or something along those lines.

Reply:
Maybe we can also set a range for our stock price in the crowdsale terms so the price will be determined when the crowdsale is finished.

Reply:
We need to be able to change this rate as and when it makes sense, given the high volatility of ETH. This is just the mechanism to allow us to change the rate.  When we change the rate is a matter for discussion however.